.github/workflows/ci.yml

```diff
@@ -29,29 +41,28 @@ jobs:
             -Dparallel-test-playback \
             -Pdev \
             -f sdk/spring/pom.xml
+    - name: Build Spring Cloud Azure Experimental libraries with Maven
```

Comment:
How about adding `sdk/spring-experimental/pom.xml` into `sdk/spring/pom.xml` dev profile?

Reply:
Let me remove it first, it will be removed when the native feature is supported.